times_aix.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package times
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+// HasChangeTime and HasBirthTime are true if and only if
+// the target OS supports them.
+const (
+	HasChangeTime = true
+	HasBirthTime  = false
+)
+
+type timespec struct {
+	atime
+	mtime
+	ctime
+	nobtime
+}
```

Comment:
Looks good according to https://www.ibm.com/support/pages/file-times-aix

Are you able to test this on an aix machine? Sadly I don't think Travis supports this os/arch so can't add automatic test coverage.

Reply:
No, Travis afaik doesn't support aix/ppc64. I was able to build it on an AIX machine with the change. I spin up a new AIX machine now and try to run example.

Reply:
I think if you're on an AIX machine with Go, you can also just run `go test` in this package and it'll run the generic tests against this code on that OS.

Reply:
Here it is:

```
$ go version
go version go1.14.12 aix/ppc64
$ go env
GO111MODULE=""
GOARCH="ppc64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="ppc64"
GOHOSTOS="aix"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="aix"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/opt/freeware/lib/golang"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/opt/freeware/lib/golang/pkg/tool/aix_ppc64"
GCCGO="gccgo"
GOPPC64="power8"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -maix64 -pthread -mcmodel=large -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build969770962=/tmp/go-build -gno-record-gcc-switches"
$ go test
PASS
ok      github.com/aklyachkin/times     0.023s
$ cd example
$ go run main.go
# FILE: /tmp/986456171
## Stat: /tmp/sym-986456171
AccessTime: 2021-03-24 20:55:52.697588 +0100 CET
ModTime: 2021-03-24 20:56:12.69759 +0100 CET
ChangeTime: 2021-03-24 20:56:02.696580503 +0100 CET

## Lstat: /tmp/sym-986456171
AccessTime: 2021-03-24 20:56:02.695280269 +0100 CET
ModTime: 2021-03-24 20:56:02.695280269 +0100 CET
ChangeTime: 2021-03-24 20:56:02.695280269 +0100 CET

# DIR: /tmp/003342542
## Stat: /tmp/sym-003342542
AccessTime: 2021-03-24 20:55:52.703817 +0100 CET
ModTime: 2021-03-24 20:56:12.703818 +0100 CET
ChangeTime: 2021-03-24 20:56:02.702891879 +0100 CET

## Lstat: /tmp/sym-003342542
AccessTime: 2021-03-24 20:56:02.702891878 +0100 CET
ModTime: 2021-03-24 20:56:02.702891878 +0100 CET
ChangeTime: 2021-03-24 20:56:02.702891878 +0100 CET
```